Tests/VauxTests/VauxTests.swift

```diff
@@ -539,6 +539,65 @@ final class VauxTests: XCTestCase {
     }
   }
   
+  func testScriptCode() {
+    let scriptID = "script"
+    func pageWithJavascript() -> HTML {
+      html {
+        body {
+          paragraph { "" }.id(scriptID)
+          script(code: "document.getElementById('\(scriptID)').innerHTML = 'Hello JavaScript!';")
+        }
+      }
+    }
+    let correctHTML = """
+        <!DOCTYPE html>
+        <html>
+          <body>
+            <p id="\(scriptID)">
+            </p>
+            <script>
+              document.getElementById('\(scriptID)').innerHTML = 'Hello JavaScript!';
+            </script>
+          </body>
+        </html>
+        """.replacingOccurrences(of: "\n", with: "")
+    let vaux = Vaux()
+    vaux.outputLocation = .file(filepath: Filepath(name: "testing", path: "/tmp/"))
+    do {
+      let rendered = try VauxTests.renderForTesting(with: vaux, html: pageWithJavascript())
+      XCTAssertEqual(rendered, correctHTML)
+    } catch let error {
+      XCTFail(error.localizedDescription)
+    }
+  }
+  
+  func testScriptFile() {
+    func buildPage() -> HTML {
+      html {
+        body {
+          script(filepath: Filepath(name: "script.js", path: ""))
```

Comment:
Is this file added to the testing folder now?

Reply:
There is no such file... Just used for the HTML generation, but we could make it also for the hack of it.

Reply:
In a future PR, let's make an actual file that does something...javascript-y. Purely for the sake of demonstration in the browser 👍